src/test/java/reposense/model/RepoConfigurationTest.java

```diff
@@ -299,4 +301,27 @@ public void repoConfig_overrideStandaloneConfig_success()
 
         TestUtil.compareRepoConfig(expectedConfig, actualConfig);
     }
+
+    @Test
+    public void repoConfig_minimalStandaloneConfig_fieldsAssignedDefaultValues() throws GitCloneException,
+            InvalidLocationException {
+        RepoConfiguration expectedConfig = new RepoConfiguration(new RepoLocation(TEST_REPO_MINIMAL_STANDALONE_CONFIG),
+                "master");
+
+        Author firstAuthor = new Author("bluein-green");
+        Author secondAuthor = new Author("jylee-git");
```

Comment:
can you test the author email, author name and display name as well?

Reply:
and its ignore glob list too.

Reply:
What do you mean?

Reply:
Hmm, actually I wanted to mention its because when we use standalone configs, the author are not important as `Author`, but rather `StandaloneAuthor`, so we should verify its value.

But on second thought, I think its fine just to simply check that it is equal to the initialized `Author`. 

Can ignore my above comments.